spec/features/admin/twitter/admin_sources_new_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe 'Admin Twitter source index' do
+  context 'user with admin privileges' do
+    scenario 'can see only sources from his organization' do
```

Comment:
here sources should be already created - at least two: one from user organization and the other one from a different organization